packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -2407,12 +2407,23 @@ describe('Checkout', () => {
                 });
             });
 
+            it('should call `trackFormInteraction` with the error information including `errorCode` when it exists', () => {
+                // Arrange
+                error.errorCode = 'OrderError';
+
+                // Act
+                wrapper.vm.handleErrorState(error);
+
+                // Assert
+                expect(trackFormInteractionSpy).toHaveBeenCalledWith({ action: 'error', error: `error_${error.errorCode}-${error.message}` });
+            });
+
             it('should call `trackFormInteraction` with the error information', () => {
                 // Act
                 wrapper.vm.handleErrorState(error);
 
                 // Assert
-                expect(trackFormInteractionSpy).toHaveBeenCalledWith({ action: 'error', error: `error_${error.message}` });
+                expect(trackFormInteractionSpy).toHaveBeenCalledWith({ action: 'error', error: `error_-${error.message}` });
```

Comment:
Nit: this looks a bit weird, since it will output something like `error_-MyMessage`.